ui/shared/components/panel/variants/ClinGenVciLink.jsx

```diff
@@ -36,13 +56,20 @@ class ClinGenVciLink extends React.PureComponent {
 
     return (
       <DataLoader contentId={hgvsc} content={alleleId || error} loading={loading} load={this.load}>
-        <a href={CLINGEN_VCI_URL} target="_blank" rel="noreferrer">In ClinGen VCI</a>
-        <br />
-        {error || (alleleId && <CopyToClipboardButton text={alleleId} />)}
+        <ClingenInfo alleleId={alleleId} error={error} />
       </DataLoader>
     )
   }
 
 }
 
+const ClinGenVciLink = ({ CAID, hgvsc }) => (
+  CAID ? <ClingenInfo alleleId={CAID} /> : <LoadedClingenVciLink alleleId={CAID} hgvsc={hgvsc} />
```

Comment:
If there is a `CAID` this conditional will render the `ClingenInfo` so the `alleleId={CAID}` for the `LoadedClingenVciLink` is always going to be undefined, so theres no need have that component accept that prop in the first place

Reply:
you're right, good catch